clients/web/src/views/layout/HeaderView.js

```diff
@@ -17,7 +17,9 @@ var LayoutHeaderView = View.extend({
         }
     },
 
-    initialize: function () {
+    initialize: function (settings) {
+        this.brandName = settings.brandName || null;
```

Comment:
This default should be `'Girder'`, as `null` will not render cleanly, and not showing the dependent elements isn't an option in this case.